README.md

```diff
@@ -182,6 +182,33 @@ If one of the allocators starts setting the withdraw queue and/or supply queue t
 
 ## Getting Started
 
+### Installation
```

Comment:
should we specify that this is for the "pkg" part?

Reply:
We should also includes the contracts to npm

Reply:
We should wait for the need to be asked by integrators before pushing contracts in the package imo